core/constants.py

```diff
@@ -2,3 +2,5 @@
 
 YETI_ROOT = path.normpath(path.join(path.dirname(path.abspath(__file__)), ".."))
 STORAGE_ROOT = path.join(YETI_ROOT, "storage")
+DB_VERSION = 0
```

Comment:
I would remove DB_VERSION and get it automatically by looking at the highest migration.